pyam/utils.py

```diff
@@ -132,6 +132,24 @@ def format_data(df, **kwargs):
     if isinstance(df, pd.Series):
         df = df.to_frame()
 
+    # Check for R-style year columns, converting where necessary
```

Comment:
suggest to move that function out of `format_data()` - the function is already too long...

Reply:
But it is also the catch-all function for saying "here is an input data set, make it into a `pyam.IamDataFrame`. I agree we could break `format_data` into multiple functions that all get called, but I don't think that's in the scope of this PR. Happy to make an issue though!